docs/instructions.adoc

```diff
@@ -74,9 +74,9 @@ If you wish to use {uri-oci-kms}[OCI KMS] to encrypt Kubernetes secrets, the fol
 ** {uri-oci-manage-policies}[manage policies in root tenancy]
 * link:#adding-the-bastion-host[bastion must be enabled]
 * link:#enabling-instance_principal-on-the-operator-host[operator instance_principal must be enabled]
-
 * use_cluster_encryption must be set to _true_
```

Comment:
Please enclose true in between `` instead of italic so it renders as `true`

Reply:
all comments in this file have _true_ without `` ....